src/fuse_scan.cpp

```diff
@@ -464,6 +472,63 @@ void FusedScan::processPointCloud(sensor_msgs::PointCloud2& cloud_msg, sensor_ms
     projector_.transformLaserScanToPointCloud(base_link_, output, cloud_processed, tflistener_);
   }
 
+void FusedScan::filterAngle(sensor_msgs::PointCloud2& cloud_msg, sensor_msgs::PointCloud& cloud_processed, int lidar_no_){
```

Comment:
is lidar_no_ a private variable? if it is not a private variable, then there should not be any leading underscore

Reply:
The function is a public member of the class, so should I rename it as lidar_no?